src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -90,10 +118,26 @@ public int countImages() {
     return images.size();
   }
 
+  public int countHeaders() {
+    return headers.size();
```

Comment:
Depend on what you are interested with, if this is count of visible `headers`, then wait for at least one visible element (this might indicated component is rendered), if you want to count headers in DOM - leave like it is. If you choose "visible" add JavaDoc with explanation